### PR TITLE
Fix UWP and commands support

### LINKED ISSUE #4320: EF does not install and build on UWP
Since dropping the "netcore50" TFM, we have not been able to install and build EF on UWP. Tracking progress on external blocking issues here.

- [x] EntityFramework.Sqlite - Fixed with #4343
- [ ] EntityFramework.Commands

### PR DESCRIPTION
**Pull request by natemcmaster**
Fix #4320 

cc @bricelam

### CODE REVIEW

File: src/EntityFramework.Commands/build/netcore50/EntityFramework.Commands.targets [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Target Name="_CopyCommandsLibrary"
+    AfterTargets="Build"
```
Comment (bricelam) [inline src/EntityFramework.Commands/build/netcore50/EntityFramework.Commands.targets]: Using the `BuildDependsOn` property is more idiomatic.

**Comment by natemcmaster** [comment]
Rebased on updated.

File: src/Microsoft.EntityFrameworkCore.Commands/build/netcore50/Microsoft.EntityFrameworkCore.Commands.targets [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+      $(BuildDependsOn);
+      _CopyCommandsLibrary
+    </BuildDependsOn>
+  </PropertyGroup>
+  <Target Name="_CopyCommandsLibrary"
+    Condition="'$(UseDotNetNativeToolchain)' != 'true'">
+    <ItemGroup>
+      <_CommandsLibrary Include="$(MSBuildThisFileDirectory)..\..\lib\net451\*.dll" />
+    </ItemGroup>
+    <Copy SourceFiles="%(_CommandsLibrary.FullPath)"
```
Comment (bricelam) [inline src/Microsoft.EntityFrameworkCore.Commands/build/netcore50/Microsoft.EntityFrameworkCore.Commands.targets]: Just `@(_CommandsLibrary)`... Actually, just hard-code `$(MSBuildThisFileDirectory)..\..\lib\net451\Microsoft.EntityFrameworkCore.Commands.dll` here and get rid of the item group.

File: src/Microsoft.EntityFrameworkCore.Commands/build/netcore50/Microsoft.EntityFrameworkCore.Commands.targets [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+          DestinationFolder="$(OutputPath)" />
```
Comment (bricelam) [inline src/Microsoft.EntityFrameworkCore.Commands/build/netcore50/Microsoft.EntityFrameworkCore.Commands.targets]: I think `$(TargetDir)` might be more reliable

Comment (bricelam) [inline src/Microsoft.EntityFrameworkCore.Commands/build/netcore50/Microsoft.EntityFrameworkCore.Commands.targets]: (`$(OutputPath)` could be relative)

**Comment by bricelam** [comment]
:shipit:

**Pull request merged**